docs/digitalocean-deployment-guide.md

```
@@ -0,0 +1,585 @@
+# List all droplets with their IPs
+doctl compute droplet list --format Name,PublicIPv4
+
+# Get specific droplet info
+doctl compute droplet get <droplet-id> --format Name,PublicIPv4
+```
+
+## Step 3: Construct Your Node Endpoint URL
+
+Once you have your public IP address, your node endpoint follows this format:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)